scrnaseq_hto.Rmd

```diff
@@ -179,7 +285,86 @@ p = p1 + p2 +
 p
 ```
 
-# Visualisation of raw and normalised HTO data {.tabset}
+```{r hto_demux_debug, eval=FALSE, include=FALSE}
```

Comment:
Mmmh .... does it make sense to have this kind of debugging code in the script? If HTODemux changes, this is still old ...

Reply:
Removed it